### ISSUE TITLE
"You do not have keys to this vehicle" after changing plates

### ISSUE THREAD

**Issue by lildiznick** [zVoxty/qb-car-license-plate#2]
Latest version of qbCore.  After changing plates, player gets a "you do not have keys to this vehicle" notification.  Player has to lock pick vehicle and it begins working normally after that.  Server uses qb-vehiclekeys, and updated it as well.

**Comment by PhysiksTV** [comment]
Change your NetEvent to this:

RegisterNetEvent('clp:client:ResetVehicleOwner', function(plate)
    TriggerEvent("vehiclekeys:client:SetOwner", plate)
	QBCore.Functions.Notify('YOU WILL NOT BE ABLE TO GARAGE WITH FAKE PLATE', 'error')
end)

This works so that crims cant garage a vehicle with fake plates.  The car must be HIDDEN after nefarious activities

**Comment by ZuluNiner** [comment]
This doesn't actually work with QB-Core as expected or with ps-mdt. The fake plate still gets registered back to the individual (because of the SetOwner callback) so isn't really fake and can still be tracked back to the individual. Also you CAN store the car even with a fake plate installed.

**Comment by PhysiksTV** [comment]
Yes but it will reset the plate, I rewrote it so you can change the plate
without the database linking player
